final-task/src/styles/header.scss

```diff
@@ -64,6 +68,11 @@
     color: $primary;
     transition: 0.3s;
     border-radius: 5px;
+    &_cart {
+      &_quantity {
+        font-size: 12px;
+      }
+    }
```

Comment:
I think that placing the indicator and button in one line is a bad move, since any change causes the entire right side of the header to shake. It's better to put it in absolute position and on top of the icon.
![image](https://github.com/ksarise/eCommerce-Application/assets/18985111/c6ab5967-1c81-47fd-b9be-a6225464b66d)

```suggestion
&_cart {
      position: relative;
      display: inline-block;
      &_quantity {
        width: 20px;
        height: 20px;
        top: -9px;
        right: 5px;
        position: absolute;
        border: 1px solid #fff;
        display: flex;
        align-items: center;
        justify-content: center;
        border-radius: 100%;
        background-color: $primary;
        color: #fff;
        z-index: 1;
      }
    }
```

Reply:
Fixed